api_test.go

```diff
@@ -327,3 +330,26 @@ func TestRandomSeed(t *testing.T) {
 		t.Errorf("Calling RandomSeed with different seeds should produce different sequence of numbers")
 	}
 }
+
+func TestProc(t *testing.T) {
```

Comment:
this is just an example to demonstrate how someone can start and stop the Proc.run inside a test